bin/main.dart

```diff
@@ -11,7 +11,7 @@ import 'package:pana/pana.dart';
 final _gray = '\u001b[1;30m';
 final _none = '\u001b[0m';
 
-String gray(text) => "$_gray$text$_none";
```

Comment:
Just add a flag, right?

Or read an environment var?

I like having another color – it lets me distinguish log output from "other" print/stdout/stderr